src/components/ConfirmDialog.tsx

```diff
@@ -30,7 +30,7 @@ export const ConfirmDialog = ({
   return (
     <Dialog open={open} onClose={!ignoreBackdropClick ? onCancel : undefined} data-testid={dialogDataTestId}>
       <DialogTitle>{title}</DialogTitle>
-      <DialogContent>{children}</DialogContent>
+      <DialogContent sx={{ bgcolor: 'secondary.light' }}>{children}</DialogContent>
```

Comment:
får ikke DialogHeader og DialogActions ulik farge nå?

Reply:
Ser greit ut her: 
![image](https://user-images.githubusercontent.com/53341294/203088632-dbfd442d-35e6-4528-a921-867e2268098f.png)

Reply:
ah, det er kanskje default for hele Dialog da siden den bruker Paper?
I så fall kan bgcolor fjernes her